app/src/modules/docs/index.ts

```diff
@@ -10,10 +10,12 @@ export default defineModule({
 	icon: 'help_outline',
 	routes: [
 		{
+			name: 'docs-routes',
 			path: '',
 			component: StaticDocs,
 			children: [
 				{
+					name: 'docs-app-overview',
```

Comment:
```suggestion
					name: 'docs-redirect-overview',
```

I believe there's a duplicated route name from the following block, resulting in the issue of non-redirection.
 
https://github.com/directus/directus/blob/fe300a78f4bd8d402a42176c8d475c15a0e00f26/app/src/modules/docs/index.ts#L22

## Before

https://user-images.githubusercontent.com/26413686/197943186-1dc50aeb-5888-4c5e-b10d-8c5021bb5539.mov

## After

https://user-images.githubusercontent.com/26413686/197943212-9036e670-c0a5-4f25-8162-3b479a2e1acd.mov

Reply:
You're right! Somehow forgot that it's a redirect after naming `docs-routes` 😄 Fixed it.